packages/relay-service/lib/payloads.js

```diff
@@ -0,0 +1,58 @@
+import VerifiableCredential from '@docknetwork/sdk/verifiable-credential';
+import {cryptoWaitReady} from '@polkadot/util-crypto';
+import {getKeypairFromDoc} from '@docknetwork/universal-wallet/methods/keypairs';
+import {getSuiteFromKeyDoc} from '@docknetwork/sdk/utils/vc/helpers';
+
+// 1 year
+const DEFAULT_EXPIRATION = 86400 * 1000 * 365;
+
+export async function generatePayload(keyPairDoc, subject) {
+  await cryptoWaitReady();
+
+  const cred = new VerifiableCredential('dock:relay');
+  cred.setContext([
+    'https://www.w3.org/2018/credentials/v1',
+    {
+      dk: 'https://ld.dock.io/credentials#',
+      RelayAuthCredential: 'dk:RelayAuthCredential',
+      limit: 'dk:limit',
+      to: 'dk:to',
+      msg: 'dk:msg',
+    },
+  ]);
+
+  cred.setIssuanceDate(new Date().toISOString());
+  cred.setExpirationDate(
+    new Date(Date.now() + DEFAULT_EXPIRATION).toISOString(),
```

Comment:
would it make sense to have to renew this more frequently?

Reply:
in a similar vein, have we given any thought to de-authorizing the connection in general?

Reply:
I can handle it during this week